tests/wpunit/Tribe/JSON_LD/AbstractTest.php

```diff
@@ -0,0 +1,176 @@
+<?php
+namespace Tribe\JSON_LD;
+
+require_once codecept_data_dir( 'classes/Tribe__JSON_LD__Test_Class.php' );
+
+use Tribe__JSON_LD__Test_Class as Jsonld;
+
+class AbstractTest extends \Codeception\TestCase\WPTestCase {
```

Comment:
OK, just feels weird to not follow our own coding standards for tests. `Tribe__Template_Checker_Test` and `Logging_Test` aren't camelcased but seem to work ok?